server/odc-service/src/main/java/com/oceanbase/odc/service/sqlcheck/rule/BaseAffectedRowsExceedLimit.java

```diff
@@ -63,6 +65,69 @@ public List<CheckViolation> check(@NonNull Statement statement, @NonNull SqlChec
         return Collections.emptyList();
     }
 
+    /**
+     * OB execute 'explain' statement
+     *
+     * @param originalSql target sql
+     * @param jdbcOperations jdbc Object
+     * @return affected rows
+     */
+    public long getOBAffectedRows(String originalSql, JdbcOperations jdbcOperations) {
```

Comment:
Does this method work for both ob oracle, oracle and ob mysql mode?

Reply:
just for ob mysql and ob oracle